gephi/gephi-plugins
# Seadragon plugin

## New plugin or plugin update?


## What is the purpose of this plugin?
Able to see large map with opensea dragon.
  
## How to test your plugin in Gephi?

1. Open Gephi, get a Spatialized Graph
2. File >> Export >> Seadragon Web ...
3. Export
4. Check that file are present in the exported directory
5. Use of webserver (``python -m http.server` is enough) to make sure it works

## Checklist before submission

* [X] Did you write unit tests to test your plugin functionalities?
